### DISCUSSION TITLE
I’ve already minted my governance token, will I still be able to use Astro?

### DISCUSSION THREAD

**Discussion by 0xQuasark** [near-daos/astro-ui#607] [Q&A] [answered]
I’ve already minted my governance token, will I still be able to use Astro?

**Comment by 0xQuasark** [comment, marked as answer]
Absolutely! You can use the [Rainbow Bridge](https://rainbowbridge.app/transfer) to bring in assets from other chains.

**Comment by starpause** [comment]
In addition to bridging assets from other chains it's possible to adopt any [NEP141](https://nomicon.io/Standards/FungibleToken/Core) as a governance token in AstroDAO. You can make your own NEP141 FT quick and easy at https://tkn.farm/

**Reply by shkfnly** [reply]
I don't see how to adopt a governance token in AstroDao can you point me in the right direction?

**Reply by shkfnly** [reply]
@starpause
